spec/domain/command_handlers/game_command_handler_spec.rb

```diff
@@ -116,231 +137,335 @@
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_right(160)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_right(180)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_right(200)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_right(220)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_right(240)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_right(260)),
+          InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_right(280)),
+          InstructionCardDealtEvent.new(id, steven, InstructionCard.rotate_right(300)),
+        )
+      end
+    end
 
-        before do
-          given_events(
-            GameCreatedEvent.new(id, GameState::LOBBYING, bob),
-            PlayerJoinedGameEvent.new(id, bob),
-            GameStartedEvent.new(id, GameState::RUNNING, deck, tiles),
-            SpawnPlacedEvent.new(id, bob, GameUnit.new(0, 0, GameUnit::DOWN)),
-            GoalPlacedEvent.new(id, Goal.new(0, 0, 1)),
-            RobotSpawnedEvent.new(id, bob, GameUnit.new(0, 0, GameUnit::DOWN)),
-            GameRoundStartedEvent.new(
-              id, GameRound.new(1), {'bob' => []}, {'bob' => []}
-            ),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(10)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(20)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(30)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(40)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(50)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.u_turn(60)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(70)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(90)),
-            InstructionCardDealtEvent.new(id, bob, InstructionCard.rotate_left(110)),
-            RobotProgrammedEvent.new(id, bob, [
-              InstructionCard.u_turn(10),
-              InstructionCard.u_turn(20),
-              InstructionCard.u_turn(30),
-              InstructionCard.u_turn(40),
-              InstructionCard.u_turn(50)
-            ]),
-            AllRobotsProgrammedEvent.new(id)
-          )
-        end
+    context "given a winning scenario" do
+      before do
+        given_events(
+          GameCreatedEvent.new(id, GameState::LOBBYING, bob),
+          PlayerJoinedGameEvent.new(id, bob),
+          GameStartedEvent.new(id, GameState::RUNNING, deck, board),
+          SpawnPlacedEvent.new(id, bob, GameUnit.new(0, 0, GameUnit::DOWN)),
+          GoalPlacedEvent.new(id, Goal.new(0, 0, 1)),
+          RobotSpawnedEvent.new(id, bob, GameUnit.new(0, 0, GameUnit::DOWN)),
+          GameRoundStartedEvent.new(id, GameRound.new(1), {"bob" => []}, {"bob" => []}),
```

Comment:
Line is too long. [88/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.